### PR TITLE
Update homecategoriez-boilerplate.tpl

### PR DESCRIPTION
**Pull request by NKoonen**
This correctly displays the images for Prestashop 1.7.X

### CODE REVIEW

File: views/templates/homecategoriez-boilerplate.tpl [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {**
  * Home Categories Block: module for PrestaShop.
  *
- * @author    Maksim T. <zapalm@yandex.com>
+ * @author    Maksim T. <zapalm>
```
Comment (zapalm) [inline views/templates/homecategoriez-boilerplate.tpl:4]: Correct is `Maksim T. <zapalm@yandex.com>`

File: views/templates/homecategoriez-boilerplate.tpl [hunk 2]
```diff
@@ -13,24 +13,22 @@
     <ul class="row">
         {foreach from=$categories item=category name=homeCategory}
             {assign var='categoryLink' value=$link->getcategoryLink($category->id_category, $category->link_rewrite)}
-            {assign var='imageLink' value=$link->getCatImageLink($category->link_rewrite, $category->id_category, $pic_size_type)}
+            {assign var='imageLink' value=$link->getCatImageLink($category->link_rewrite, $category->id_image, 'category_default')|escape:'html':'UTF-8'}
```
Comment (zapalm) [inline views/templates/homecategoriez-boilerplate.tpl:16]: This is wrong. The previous code was correct.

File: views/templates/homecategoriez-boilerplate.tpl [hunk 3]
```diff
@@ -13,24 +13,22 @@
...
             <li class="col-xs-12 col-sm-6 col-lg-2">
                 <a href="{$categoryLink}" title="{$category->name|escape:html:'UTF-8'}">
-                    {if file_exists($smarty.const._PS_CAT_IMG_DIR_|cat:basename($imageLink))}
+                        {if $category->id_image}
```
Comment (zapalm) [inline views/templates/homecategoriez-boilerplate.tpl:19]: This is correct, but the strict style is preferred: `{if $category->id_image|intval > 0}`

File: views/templates/homecategoriez-boilerplate.tpl [hunk 4]
```diff
@@ -13,24 +13,22 @@
...
-                    {if file_exists($smarty.const._PS_CAT_IMG_DIR_|cat:basename($imageLink))}
+                        {if $category->id_image} 
                         <img
                             src="{$imageLink}"
                             width="{$pic_size.width}"
                             height="{$pic_size.height}"
                             alt="{$category->name|escape:html:'UTF-8'}"
-                            title="{$category->name|escape:html:'UTF-8'}"
-                        >
+                            title="{$category->name|escape:html:'UTF-8'}">
```
Comment (zapalm) [inline views/templates/homecategoriez-boilerplate.tpl:25]: There was a more friendly formatting style.

File: views/templates/homecategoriez-boilerplate.tpl [hunk 5]
```diff
@@ -13,24 +13,22 @@
...
                     {else}
                         <img
                             src="{$urls.img_cat_url|escape:'html':'UTF-8'}{$language.iso_code|escape:'html':'UTF-8'}.jpg"
                             width="{$pic_size.width}"
                             height="{$pic_size.height}"
-                            title="{l s='No image' mod='tablecombz'}"
-                        >
+                            title="{l s='No image' mod='tablecombz'}">
```
Comment (zapalm) [inline views/templates/homecategoriez-boilerplate.tpl:31]: 1. There was a more friendly formatting style.
2. There is a mistake: change `tablecombz` to `homecategoriez`

**Review by zapalm — COMMENTED** [review]
See comments.

**Comment by zapalm** [comment]
@NKoonen 
Hello, are you still want to contribute? :)

**Comment by NKoonen** [comment]
> @NKoonen
> Hello, are you still want to contribute? :)

Hi,
Yeah in the coming days i will make another version.

**Comment by zapalm** [comment]
Closed because of no activity.
The problem was resolved in this commit: https://github.com/zapalm/homecategoriez/commit/efa7ccf014959b229c8944a5494cdbdbd91c1850

**Pull request closed without merging**